src/libraries/System.Private.CoreLib/src/System/IO/FileStream.cs

```diff
@@ -397,18 +397,9 @@ public override long Position
         /// <param name="value">The byte to write to the stream.</param>
         public override void WriteByte(byte value) => _strategy.WriteByte(value);
 
-        ~FileStream()
```

Comment:
Mind explaining why this is being removed from `FileStream`? Isn't below comment saying that it is preserved for compatibility?